specklepy/objects/base.py

```diff
@@ -317,8 +324,8 @@ def units(self, value: str):
 
     def get_member_names(self) -> List[str]:
         """Get all of the property names on this object, dynamic or not"""
-        # attrs = set(self.__dict__.keys())
-        attr_dir = list(set(dir(self)) - REMOVE_FROM_DIR)
+        ignored_attributes = REMOVE_FROM_DIR.union(self._serialize_ignore)
```

Comment:
the users should actually be able to see these attrs. they are interesting to the user and ignoring them is _only_ for the benefit of serialisation and deserialisation. instead, we should do this in the deserialiser when traversing through the attributes and ignore it if it exists in `_serialize_ignore`

Reply:
Resolved in 3005e42